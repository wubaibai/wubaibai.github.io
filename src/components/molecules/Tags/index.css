.tags {
	margin-bottom: -12px;

	& > * {
		margin-right: 12px;
		margin-bottom: 12px;
	}
}
