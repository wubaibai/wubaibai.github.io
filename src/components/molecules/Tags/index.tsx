import React from 'react';

import Tag from 'components/atoms/Tag';

import style from './index.css';

interface TagsProperty {
	items: string[];
}

const Tags: React.FC<TagsProperty> = ({ items }) => (
	<div className={style.tags}>
		{items.map(item => (
			<Tag key={item}>{item}</Tag>
		))}
	</div>
);

export default Tags;
