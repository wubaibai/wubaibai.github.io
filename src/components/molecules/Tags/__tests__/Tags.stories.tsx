import React from 'react';
import { storiesOf } from '@storybook/react';

import Tags from 'components/molecules/Tags';

const stories = storiesOf('molecules/Tags', module);
const items = ['Node.js', 'HTML5', 'CSS3'];

stories.add('__interactive', () => <Tags items={items} />, { jest: 'Tags' });
