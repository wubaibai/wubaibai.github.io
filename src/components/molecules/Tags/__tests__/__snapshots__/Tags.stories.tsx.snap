// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`__interactive 1`] = `
<DocumentFragment>
  <div
    class="tags"
  >
    <div
      class="tag"
    >
      <div
        class="text"
      >
        Node.js
      </div>
      <div
        class="base"
      />
    </div>
    <div
      class="tag"
    >
      <div
        class="text"
      >
        HTML5
      </div>
      <div
        class="base"
      />
    </div>
    <div
      class="tag"
    >
      <div
        class="text"
      >
        CSS3
      </div>
      <div
        class="base"
      />
    </div>
  </div>
</DocumentFragment>
`;
